docbook/gnc-docbookx.dtd

```diff
@@ -165,9 +165,6 @@ own entity definitions to use across all of our DocBook pages. -->
 
 <!-- c) URLs of Authors -->
 <!ENTITY url-selinger "https://www.mathstat.dal.ca/~selinger/">
-<!ENTITY tit-sel-tut "Tutorial on multiple currency accounting">
-<!ENTITY tit-sel-gnc "Multiple currency accounting in GnuCash">
+<!ENTITY tit-sel-tut "&url-selinger;accounting/tutorial.html">
+<!ENTITY tit-sel-gnc "&url-selinger;accounting/gnucash.html">
```

Comment:
- Before it has been the titles—Label of the links— not the URLs
- Using entities inside of other entities is now possible?

Reply:
@fellen : Ja, aber jetzt passen die Verknüpfungen besser und bringen den interessierten Leser direkt an die richtige Stelle. Und die verschachtelte Entität funktioniert ebenfalls.

Reply:
zu 2: gute Info, danke

zu 1: Haben sie vorher auch:

Was ich nicht so schön finde:

1. Du ersetzt dieTitel der Dokumente durch ihre UR.
2. Du entfernst die Trennung von technischer URL und lokalisierbarem Titel. Pedro hat ihn z.B.—noch etwas ungeschickt—in  `<foreignphrase></foreignphrase>` gekapselt.
3. Das sind Änderungen, die besser in einem eigenen Commit zusammen mit den zugehörigen Anpassungen in allen Vorkommen in allen Sprachen durchgeführt werden.
4. Wie wäre es mit
```
<!ENTITY url-selinger "https://www.mathstat.dal.ca/~selinger/">
<!ENTITY tit-sel-tut "Tutorial on multiple currency accounting">
<!ENTITY ul-sel-tut "<ulink url="&url-selinger;accounting/tutorial.html">"&tit-sel-tut;"</ulink>">
```
und in einem der beiden letzten noch die foeignphrase einbauen.
Wenn du Bits sparen willst, kannst du "accounting/" auch noch nach oben verschieben.
- *-sel-gnc analog
- In den Texten dann die ul-sel-* verwenden.

Reply:
@fellen: mhh, da war ich wohl etwas übereifrig :(
Ich schau mir deinen Vorschlag mal an und versuche das umzusetzen.